src/format.rs

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: Apache-2.0
+
+//! Supported output formats for the resolved registry and telemetry schema
+
+use clap::ValueEnum;
+use serde::Serialize;
+
+/// Supported output formats for the resolved schema
+#[derive(Debug, Clone, ValueEnum)]
+pub(crate) enum Format {
```

Comment:
Moved `Format` in a more global location as this structure is now used from multiple sub-commands.